models/UserInterface.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Model;
+
+
+use Pimcore\Model\User\UserRoleInterface;
+
+/**
+ * @method User\Dao getDao()
```

Comment:
Why like this and not in the interface?

Reply:
You are right, this can be removed because it is already in the ModelInterface

Reply:
Try to remove this, but break phpStan checks. So have to add it again.